Makefile

```diff
@@ -234,9 +234,13 @@ else ifeq ($(CHAIN),polygon)
 APP_LOAD_PARAMS += --path "44'/60'"
 DEFINES += CHAINID_UPCASE=\"POLYGON\" CHAINID_COINNAME=\"MATIC\" CHAIN_KIND=CHAIN_KIND_POLYGON CHAIN_ID=137
 APPNAME = "Polygon"
+else ifeq ($(CHAIN),bttc)
+APP_LOAD_PARAMS += --path "44'/554'" --path "44'/60'"
```

Comment:
554' is a reserved value, according to [SLIP-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md). It was previously used by Flare Spark.

Why are you using this value?

Reply:
Yes, you are right.. my mistake.. have pushed another commit to remove this. Thanks!